### DISCUSSION TITLE
Graphql IDE support

### DISCUSSION THREAD

**Discussion by 97amarnathk** [enisdenjo/graphql-ws#114] [Q&A] [answered]
Is there any graphql GUI which I can use to test my subscriptions ?

Currently I have to write a node script using urql and graphql-ws client. 
Is there a better and an easier way to test out subscriptions ?

**Comment by n1ru4l** [comment, marked as answer]
You could simply setup your own GraphiQL on a route on your local Frontend dev server.

See https://github.com/n1ru4l/graphql-bleeding-edge-playground for an example

**Reply by momics** [reply]
I think this pull request allows you to use graphql-ws with GraphiQL as well. Haven't used it yet though..

https://github.com/graphql/graphiql/pull/1770

**Reply by n1ru4l** [reply]
yes, GraphiQL now ships with a default fetcher which is basically an abstraction over the thing I linked above :)
